src/internal/provider/protocols.go

```diff
@@ -23,7 +23,7 @@ func (p Provider) GetProtocols(manifestDownloadUrl string) ([]string, error) {
 
 	manifest, err := parseManifestContents(contents)
 	if err != nil {
-		return nil, err
+		return nil, nil
```

Comment:
Can we add the following log message?
```go
p.Logger.Warn("Ignoring invalid manifest file", slog.String("url", manifestDownloadUrl), slog.Any("err", err))
```

Reply:
a710b83069b3405d0f2d77b49f0b43849f7b7e72